typescript/infra/config/environments/mainnet2/agent.ts

```diff
@@ -1,23 +1,30 @@
-import { ALL_KEY_ROLES } from '../../../src/agents/roles';
+import { ALL_KEY_ROLES, KEY_ROLE_ENUM } from '../../../src/agents/roles';
 import { AgentConfig } from '../../../src/config';
 import {
   ConnectionType,
   GasPaymentEnforcementPolicyType,
 } from '../../../src/config/agent';
 import { Contexts } from '../../contexts';
+import { helloworldMatchingList } from '../../utils';
+import { helloWorld } from '../mainnet2/helloworld';
 
 import { MainnetChains, chainNames, environment } from './chains';
 // import { helloWorld } from './helloworld';
 import { validators } from './validators';
 
+const releaseCandidateHelloworldMatchingList = helloworldMatchingList(
+  helloWorld,
+  Contexts.ReleaseCandidate,
+);
+
 export const hyperlane: AgentConfig<MainnetChains> = {
   environment,
   namespace: environment,
   runEnv: environment,
   context: Contexts.Hyperlane,
   docker: {
     repo: 'gcr.io/abacus-labs-dev/hyperlane-agent',
-    tag: 'sha-c163fce',
+    tag: 'sha-0477ee1',
```

Comment:
jw if you ended up deploying this on mainnet?

Reply:
We rolled it back